### PR TITLE
feat: add option to generate default paths

### PR DESCRIPTION
**Pull request by mahapo**

### CODE REVIEW

File: lib/module.js [hunk 1]
```diff
@@ -2,23 +2,24 @@ const StoryblokClient = require('storyblok-js-client')
 const { pascalCase, normalizeName, isExcluded, addTrailingSlash } = require('./utils')
 const logger = require('./logger')
 
-export default async function storyblokRouterModule(moduleOptions) {
+module.exports = async function storyblokRouterModule (moduleOptions) {
```
Comment (seralichtenhahn) [inline lib/module.js]: Why is this better like that?

File: .eslintignore [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-coverage
+**/node_modules
+**/dist
+**/.nuxt
+**/coverage
 
 # Plugin
-lib/plugin.js
\ No newline at end of file
+**/plugin.js
```
Comment (seralichtenhahn) [inline .eslintignore]: why did you change this?

File: .gitignore [hunk 3]
```diff
@@ -1,59 +1,149 @@
+# Created by .ignore support plugin (hsz.mobi)
```
Comment (seralichtenhahn) [inline .gitignore]: Do you really have to rewrite the whole file? Just add the Rules that are missing.

File: lib/module.js (hunk 1 above)
Comment (mahapo) [reply lib/module.js]: don't mix import style in one file

File: .eslintignore (hunk 2 above)
Comment (mahapo) [reply .eslintignore]: copied from other nuxt-community modules

File: lib/module.js (hunk 1 above)
Comment (seralichtenhahn) [reply lib/module.js]: but: https://nuxtjs.org/guide/modules#write-a-basic-module

File: .eslintignore (hunk 2 above)
Comment (seralichtenhahn) [reply .eslintignore]: but here it makes no sense. I just want to ignore the linter on one file, not every file called `plugin.js`
